src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx

```diff
@@ -382,33 +388,23 @@ export function getBaseLevelEntryAttributes (serverId, baseDn, entryAttributesCa
   cockpit
     .spawn(cmd, { superuser: true, err: 'message' })
     .done(data => {
-      // console.log(`data = ${data}`);
-      // const lines = data.split('\n');
       // TODO: Make this configurable ( option to keep X number of characters )
-      // Meanwhile just keep only the 150K first characters.
-      const isLarge = data.length > 150000;
-      if (isLarge) {
-        data = data.substring(0, 150000);
-      }
       const lines = data.split('\n');
       lines.map(currentLine => {
         if (currentLine !== '') {
-          entryArray.push(splitAttributeValue(currentLine));
+          if (currentLine.length < 1000 || currentLine.substring(0, 9).toLowerCase().startsWith("jpegphoto")) {
+            entryArray.push(splitAttributeValue(currentLine));
+          } else {
+            const myTruncatedValue = (<div>
+                                          currentLine.substring(0, 1000)
+                                          <Label icon={<InfoCircleIcon />} color="blue" >
+                                              Attribute is too large. Output was truncated.
```

Comment:
Output is not truncated :-)  It's not displayed at all.  Maybe something like "Value is too large to display"

Reply:
Oh, right, my copy bad)
I'll fix it.